### ISSUE TITLE
Samsung 2015 TV, only livestreams are working

### ISSUE THREAD

**Issue by manderss99** [plexinc-plugins/BBC-iPlayer.bundle#31]
I have a samsung ue55js9005 tv and uses latest version of plex server with iplayer plugin.
Live streams work fine but other programs does not work, no error message is displayed, it just hangs.
Please look into this, I can support with testing if needed.

**Comment by meriko** [comment]
@manderss99 
Do you have any logs available? Try this:

1. Restart Plex Media Server
2. Try to play a program
3. Attach the com.plexapp.system.log using for example: https://gist.github.com

**Comment by manderss99** [comment]
here is log file:
https://gist.github.com/anonymous/ea0d555ab2503fdc511c <https://gist.github.com/anonymous/ea0d555ab2503fdc511c>

**Comment by meriko** [comment]
Thanks!

VOD programs utilize HLS version 4 whilst live streams utilizes HLS version 2, is this maybe a problem/limitation in the Samsung client?

**Comment by manderss99** [comment]
Seems like hls v3 is supported by 2015 tizen models.
http://www.samsungdforum.com/TizenDevtools/Spec

Is it possible to transcode the VOD programs to make it work?

Thanks

Mikael

**Comment by meriko** [comment]
Unfortunately the channel can't force a client to request a transcode. The channel returns the codecs, used protocol etc and then it's the client that makes the decision to direct play or transcode.

Can you force a transcode in the client like this? (second paragraph, preplay screen)
https://forums.plex.tv/discussion/comment/985986/#Comment_985986

**Comment by sander1** [comment]
Yep, this doesn't sound like a channel issue, but a Plex client issue...

**Comment by manderss99** [comment]
Hi
I tried to force transcoding in the client but no luck ..,

Mikael
